### PR TITLE
Issue 15: Populated Get Technical Support

### PR DESCRIPTION
**Pull request by edeediong**
Signed-off-by: Edidiong Etuk <edidiongmichael08@gmail.com>
Closes: #15

### CODE REVIEW

File: common-requests/get-technical-support.md [hunk 1]
```diff
@@ -1,8 +1,20 @@
...
+description: ""
+tags:
+ - Agent
+ - Help
+ - Support
+author: edeediong
 ---
 
-# Get Technical Support
+## Get Technical Support
```
Comment (Rene-Ch1) [inline common-requests/get-technical-support.md:12]: Chg to: Technical support

File: common-requests/get-technical-support.md [hunk 2]
```diff
@@ -1,8 +1,20 @@
...
 
+To get technical support on Open Horizon, join the LF Edge [Slack workspace](https://join.slack.com/t/lfedge/shared_invite/zt-7kavdtmq-SeyFzM2CEABBcKYGEVCgkw) if you haven't.
```
Comment (Rene-Ch1) [inline common-requests/get-technical-support.md:14]: Chg to:

For Open Horizon technical support, join the LF Edge [Slack workspace](https://join.slack.com/t/lfedge/shared_invite/zt-7kavdtmq-SeyFzM2CEABBcKYGEVCgkw).

File: common-requests/get-technical-support.md [hunk 3]
```diff
@@ -1,8 +1,20 @@
...
+Ensure you join the *#open-horizon-help* channel on LF Edge as you can post all technical and non-technical inquiries get a response from the help team.
```
Comment (Rene-Ch1) [inline common-requests/get-technical-support.md:15]: Chg to:

When you join the LF Edge *#open-horizon-help* slack channel, you can post technical and non-technical inquiries and receive help team responses.

File: common-requests/get-technical-support.md [hunk 4]
```diff
@@ -1,8 +1,20 @@
...
+
+See an example of a query below:
```
Comment (Rene-Ch1) [inline common-requests/get-technical-support.md:17]: Chg to: 

Here is an example query:

**Review by Rene-Ch1 — CHANGES_REQUESTED** [review]

**Comment by clementkng** [comment]
Hi @edeediong this LGTM, even though I wasn't assigned as a reviewer.

File: common-requests/get-technical-support.md (hunk 1 above)
Comment (edeediong) [reply common-requests/get-technical-support.md]: Done!

File: common-requests/get-technical-support.md (hunk 2 above)
Comment (edeediong) [reply common-requests/get-technical-support.md]: Done.

File: common-requests/get-technical-support.md (hunk 3 above)
Comment (edeediong) [reply common-requests/get-technical-support.md]: Done

File: common-requests/get-technical-support.md (hunk 4 above)
Comment (edeediong) [reply common-requests/get-technical-support.md]: Done!

File: common-requests/get-technical-support.md [hunk 5]
```diff
@@ -1,8 +1,24 @@
...
 
+Open Horizon uses [Slack](https://slack.com/help/articles/115004071768-What-is-Slack-) for internal communication.
+
+For Open Horizon technical support, join the LF Edge [Slack workspace](https://join.slack.com/t/lfedge/shared_invite/zt-7kavdtmq-SeyFzM2CEABBcKYGEVCgkw) if you haven't.
+When you join the LF Edge *#open-horizon-help* slack channel, you can post technical and non-technical inquiries receive help team responses.
+
+Here is an example query:
+
+```Text
+I have the OH Management Hub on one machine and the Agent on a Raspberry Pi.
```
Comment (joewxboy) [inline common-requests/get-technical-support.md:22]: Let's add more detail:

`a Raspberry Pi` change to `a Raspberry Pi 3 B+ running Raspberry Pi OS 32-bit Lite 2020-08-20 for armhf`

Comment (edeediong) [reply common-requests/get-technical-support.md:22]: Alright I'd make these changes

**Review by joewxboy — DISMISSED** [review]

File: common-requests/get-technical-support.md [hunk 6]
```diff
@@ -1,8 +1,24 @@
...
+ - Support
+author: edeediong
 ---
 
-# Get Technical Support
+## Technical Support
 
+Open Horizon uses [Slack](https://slack.com/help/articles/115004071768-What-is-Slack-) for internal communication.
+
+For Open Horizon technical support, join the LF Edge [Slack workspace](https://join.slack.com/t/lfedge/shared_invite/zt-7kavdtmq-SeyFzM2CEABBcKYGEVCgkw) if you haven't.
```
Comment (Rene-Ch1) [inline common-requests/get-technical-support.md:16]: Suggest deleting: 

"if you haven't" in this sentence. That string is not really needed.

File: common-requests/get-technical-support.md [hunk 7]
```diff
@@ -1,8 +1,24 @@
...
+When you join the LF Edge *#open-horizon-help* slack channel, you can post technical and non-technical inquiries receive help team responses.
```
Comment (Rene-Ch1) [inline common-requests/get-technical-support.md:17]: Add "and" here:

inquiries and receive help team responses

**Review by Rene-Ch1 — CHANGES_REQUESTED** [review]

File: common-requests/get-technical-support.md (hunk 6 above)
Comment (edeediong) [reply common-requests/get-technical-support.md]: Changes applied

File: common-requests/get-technical-support.md (hunk 7 above)
Comment (edeediong) [reply common-requests/get-technical-support.md]: I've applied changes

**Review by Rene-Ch1 — APPROVED** [review]

**Review by joewxboy — APPROVED** [review]
Looks good.

**Comment by joewxboy** [comment]
This closes issue 15

**Pull request merged**